lib/services/rpcbind.pm

```diff
@@ -33,8 +33,9 @@ sub check_install {
 }
 
 sub config_service {
-    type_string("echo '/mnt *(ro,root_squash,sync,no_subtree_check)' >> /etc/exports\n");
-    type_string("echo 'nfs is working' > /mnt/test\n");
+    assert_script_run("mkdir -p /rpcbindtest");
+    type_string("echo '/rpcbindtest *(ro,root_squash,sync,no_subtree_check)' >> /etc/exports\n");
```

Comment:
OT: Why is `type_string` used here ? It will make failure just less obvious

Reply:
I don't understand either.

Reply:
Please use `assert_script_run` also there.

Reply:
I'd also vote for `assert_script_run` here.

Reply:
Did so and restarted VRs.